### ISSUE TITLE
BookCModel and PDFs

### ISSUE THREAD

**Issue by kromabiles** [mnylc/islandora_multi_importer#119]
Hello Diego,

As you know, our IR is currently exploring ways to use the IMI to ingest/create BookCModel objects from PDFs. C

**Comment by DiegoPino** [comment]
@kromabiles great. Following up here. A few questions about this:
- You good with IMI extracting PDF into TIFFs? Or do you want IMI to use the same config/Book reader already uses?
- How do we deal with page level metadata? I see we have two options (both could be implemented)
  - You actually create the rows for each page, and i find some clever UI way of letting IMI know it should only fill the OBJ from the parent column PDF (extracted as TIFF). This also means if you add 10 instead of, e.g 100 pages that the PDF contains it would only ingest 10.
 - You add nothing. If so, then IMI will create the most basic Metadata for you, basically just the title and the page number.

Processing of this would need to actually happen during ingest (batch) or it would be just too slow... we need to test.

What is your largest PDF around there? 

Secondly. I will also enable a Digital object with the same directly on play.archipelago.nyc so we can test performance and compare.

Thanks!

**Comment by kromabiles** [comment]
@DiegoPino Yes, extracting PDFs into TIFFs would be great. Our book collections don't have any page level metadata - all structured as single object description. :/

Our largest PDF is about 3GB and consists of 93 pages (yearbook). 

Seeing Archipelago in action sounds exciting! :)

**Comment by DiegoPino** [comment]
Excellent. I will start planning. Will probably borrow book module settings, but i feel i should go TIFF first and the compress to JP2 if needed. I just tested a JP2 generated by islandora (core) and it was 25 Mbytes in size, same TIFF was 10 Mbytes which was a little bit annoying!

**Comment by DiegoPino** [comment]
@kromabiles sorry for the slowness, i have a solution! But requires some testing, planning. Give me the end of the week to enable in our sandbox and i give you credentials there. Will also copy your Templates and prepare a spreadsheet testcase, but even better if you have a few PDFs in a zip and a demo spreadsheet around

**Comment by kromabiles** [comment]
No worries! Thanks, Diego - files are too big to attach here, so I'll send them over to you via email.
